lib/stringtheory/src/lib.rs

```diff
@@ -290,23 +250,23 @@ impl fmt::Display for MetaString {
     }
 }
 
-/// A string that can be inlined up to 31 bytes.
+/// A string that can be inlined up to 23 bytes.
 #[derive(Clone)]
 struct InlinedString {
-    data: [u8; 32],
+    data: [u8; 24],
```

Comment:
Note to self: we might want to make the array size a const generic parameter in the future, and that way some of the logic below -- knowing that we can encode the string size in `u8`, etc -- can be made entirely const, making it all a little more safe in the process.